### ISSUE TITLE
[Sariras Recipes] Crafting recipes are redundant?/Produce lid in recipe

### ISSUE THREAD

**Issue by UnlimatedStone9** [An-Sar/PrimalCore#313]
I made a bunch of mud bricks to craft a sariras lid and a sariras, I made the lid and then afterwards made the sariras, upon placing it, it had a lid on it already making the crafting of the lid redundant, unless a player loses it, this looks like an accidental recipe that should instead craft one without a lid

**Comment by An-Sar** [comment]
This is working as intended, since the lid is required to use a sariras it seemed necessary to be able to replace lost or misplaced lids. 

i suppose the real question is if the base sariras should or should not come with a lid included?

**Comment by UnlimatedStone9** [comment]
Yes, that was my question, the sariras comes with a lid upon crafting, I assume you read is as, why is the lid craftable?
I'd rather it craft the sariras only and not a free lid, it seems a little wrong to me and might not be noticeable, which would lead to times where people craft a spare lid, as I did before

**Comment by UnlimatedStone9** [comment]
I recommend you make it craft a sariras without a lid
